packages/twenty-front/src/modules/ui/input/components/Checkbox.tsx

```diff
@@ -165,7 +164,7 @@ export const Checkbox = ({
     setIsInternalChecked(event.target.checked ?? false);
   };
 
-  const checkboxId = 'checkbox' + v4();
+  const checkboxId = React.useId();
```

Comment:
import from react?

Reply:
The whole component imports hooks from the React global export, so I did the same.